src/OpenTelemetry.Instrumentation.Process/ProcessMetrics.cs

```diff
@@ -25,21 +24,22 @@ internal class ProcessMetrics
     internal static readonly AssemblyName AssemblyName = typeof(ProcessMetrics).Assembly.GetName();
     internal static readonly Meter MeterInstance = new(AssemblyName.Name, AssemblyName.Version.ToString());
 
-    private const string MetricPrefix = "process.runtime.dotnet.";
+    private const string MetricPrefix = "process.";
+    private static readonly System.Diagnostics.Process CurrentProcess = System.Diagnostics.Process.GetCurrentProcess();
```

Comment:
using System.Diagnostics?

Reply:
Compiler seems to be confused between System.Diagnostics.Process vs OpenTelemetry.Instrumentation.Process if we use using the System.Diagnostics.

Reply:
Ah I see, thanks! In this way the [alias](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/595#discussion_r954405557) does seem to be a good fit.